### DISCUSSION TITLE
REST API to create poll

### DISCUSSION THREAD

**Discussion by marc0l92** [lukevella/rallly#406] [Ideas]
Hello,
could it be possible to have a set of REST API to create new poll.

I would like to create a new poll every week and share it with my friends so we can define when we are free to play together the next week.

**Comment by lukevella** [comment]
It's possible but not likely. The demand for this is too small to justify the effort needed to implement and maintain an API for developers. I think a more suitable solution for your problem is a recurring meeting poll which has a much wider appeal. Can't say exactly when that will be available though as it is quite far down the line.
